documentation/library_guide/extension_api.rst

```diff
@@ -313,3 +313,67 @@ Example of Range-based API usage
 
         copy(oneapi::dpl::execution::dpcpp_default, view, range_res);
     }
+
+Async API
+--------------------------
+
+Functions defined in the STL ``<algorithm>`` or ``<numeric>`` headers are traditionally blocking. oneDPL extends the functionality of C++17 parallel algorithms by providing asynchronous algorithm with non-blocking behavior. This experimental feature enables expressing concurrent control flow by building dependency chains and interleaving algorithm calls, as well as interoperability with SYCL kernels. 
+
+The current implementation for async algorithms is limited to DPC++ Execution Policies.
+All functionality described below is available in ``oneapi::dpl::experimental`` namespace.
+
+The following async algorithms are currently supported:
+
+* ``copy_async``
+* ``fill_async``
+* ``for_each_async``
+* ``reduce_async``
+* ``transform_async``
+* ``transform_reduce_async``
+* ``sort_async``
+
+All interfaces listed above are a subset of C++17 STL algorithms
```

Comment:
Please add the bold below:

All **the** interfaces listed above are a subset of C++17 STL algorithms**,**